src/namespace.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2014 WebFilings, LLC
```

Comment:
I was wondering that the other day too. Maybe a separate task to update all files.